src/EnergyPlus/HVACControllers.cc

```diff
@@ -1145,30 +1159,30 @@ namespace HVACControllers {
 				//       - Node(SensedNode)%HumRatMax
 				//       - Node(SensedNode)%Temp
 				//       - Node(SensedNode)%HumRat
-				if ( Node( SensedNode ).HumRatMax > 0 ) {
+				if ( ( Node( SensedNode ).HumRatMax > 0 ) && ( Node( SensedNode ).HumRat > Node( SensedNode ).HumRatMax ) ) {
 					// Setpoint can only be computed when the sensed node temperature is evaluated at the max
 					// actuated value for the dual humidity ratio / temperature strategy.
 					// See routine CalcSimpleController() for the sequence of operations.
-					if ( ControllerProps( ControlNum ).NextActuatedValue == RootFinders( ControlNum ).MaxPoint.X ) {
```

Comment:
This comment should be revised to describe revised code. I'll do that after all comments are received.